schema.json

```diff
@@ -0,0 +1,37 @@
+
+{
+    "database_name" : "Check-in",
+
+    "Tables" : [
+        {
+            "name" : "Reservations",
```

Comment:
I have a remark here. Those keys the name, columns, ...etc are the same as those in ```SchemaKeys``` so I think they should be capiltalised as they are in the ```SchemaKeys```.

Reply:
TABLE_NAME = “name”

the variable name is capitalize but is value same as at json